c_glib/arrow-glib/compute.cpp

```diff
@@ -1412,6 +1412,42 @@ garrow_array_filter(GArrowArray *array,
   }
 }
 
+/**
+ * garrow_array_isin:
+ * @left: A left hand side #GArrowArray.
+ * @right: A right hand side #GArrowArray.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: (nullable) (transfer full): The #GArrowBooleanArray
+ *   showing whether each element in the left array is contained
+ *   in right array.
+ *
+ * Since: 0.15.0
+ */
+GArrowBooleanArray *
+garrow_array_isin(GArrowArray *left,
```

Comment:
Could you rename to `garrow_array_is_in()`?

Reply:
I've renamed it.